lexos/static/js/scripts_manage.js

```diff
@@ -1,803 +1,906 @@
+                handleSelectButtons(table.rows().ids().length, table.rows({selected: true}).ids().length)
+                $('.fa-folder-open-o')[0].dataset.originalTitle = 'You have 1 active document(s)'
+                // toggleActiveDocsIcon();
+                $('#edit-modal').modal('hide')
+                $('#edit-form').remove()
+            })
+        .fail(
+            function (jqXHR, textStatus, errorThrown) {
+                $('#error-modal .modal-body').html('Lexos could not merge the requested documents or could not save the merged document.')
+                $('#error-modal').modal()
+                $('#edit-modal').modal('hide')
+                console.log('bad: ' + textStatus + ': ' + errorThrown)
+            })
+
 }
+
 /* #### END OF mergeDocuments() #### */
 
 /* #### saveMultiple() #### */
+
 // Helper function saves value in edit dialog and updates table for multiple rows
-function saveMultiple(row_ids, column, value) {
-  // Prepare data and request
-  url = '/setClassSelected'
-  data = JSON.stringify([row_ids, value])
-
-  // Do Ajax
-  $.ajax({
-    type: 'POST',
-    url: url,
-    data: data,
-    contentType: 'application/json;charset=UTF-8',
-    cache: false,
-    success: function (response) {
-      row_ids = JSON.parse(response)
-      // Update the UI
-      var reloadPage = false
-      $.each(row_ids, function (i) {
-        id = '#' + row_ids[i]
-        $(id).find('td:eq(2)').text(value)
-        if ($(id).length == 0) {
-          reloadPage = true
-        }
-      })
-      $('#edit-modal').modal('hide')
-      $('#edit-form').remove()
-      // Ugly hack to make sure rows are updated across table pages
-      if (reloadPage == true) {
-        window.location.reload()
-      } else {
-        toggleActiveDocsIcon()
-        table.draw()
-      }
-    },
-    error: function (jqXHR, textStatus, errorThrown) {
-      $('#error-modal .modal-body').html('Lexos could not update the class of the requested documents.')
-      $('#error-modal').modal()
-      $('#delete-modal').modal('hide')
-      console.log('bad: ' + textStatus + ': ' + errorThrown)
-    }
-  })
+function sendAjaxRequestSaveMultiple (url, data) {
+    return $.ajax({
+        type: 'POST',
+        url: url,
+        data: data,
+        contentType: 'application/json;charset=UTF-8',
+        cache: false
+    })
 }
+
+function saveMultiple (row_ids, column, value) {
+    // Prepare data and request
+    console.log('inside the save')
+    url = '/setClassSelected'
+    data = JSON.stringify([row_ids, value])
+    // Do Ajax
+    sendAjaxRequestSaveMultiple(url, data)
+        .done(
+            function (response) {
+                console.log('save multiple: ' + response)
+                row_ids = JSON.parse(response)
+                // Update the UI
+                var reloadPage = false
+                $.each(row_ids, function (i) {
+                    id = '#' + row_ids[i]
```

Comment:
Never use `'str'+something`, always use template literal when you need to put some value in a string